Makefile

```
@@ -10,10 +10,10 @@ all: dollarskip
 install: dollarskip
 	$(STRIP) dollarskip
 	install -d $(DESTDIR)$(PREFIX)/bin
-	install -m 755 dollarskip $(DESTDIR)$(PREFIX)/bin/\$
+	install -m 755 dollarskip $(DESTDIR)$(PREFIX)/bin/\$$
 
 uninstall:
-	rm -f $(DESTDIR)$(PREFIX)/bin/\$
+	rm -f $(DESTDIR)$(PREFIX)/bin/\$$
```

```suggestion
	rm -f $(DESTDIR)$(PREFIX)/bin/$$
```

same.
tested with GNU make 3.81 on MacOS.